lib/services/presets.go

```diff
@@ -561,6 +562,111 @@ func NewSystemOktaRequesterRole() types.Role {
 	return role
 }
 
+// NewSystemIdentityCenterAccessRole creates a role that allows access to AWS
+// IdentityCenter resources via Access Requests
+func NewSystemIdentityCenterAccessRole() types.Role {
+	if modules.GetModules().BuildType() != modules.BuildEnterprise {
+		return nil
+	}
+
+	return &types.RoleV6{
+		Kind:    types.KindRole,
+		Version: types.V7,
+		Metadata: types.Metadata{
+			Name:        teleport.SystemIdentityCenterAccessRoleName,
+			Namespace:   apidefaults.Namespace,
+			Description: "Access AWS IdentityCenter resources",
+			Labels: map[string]string{
+				types.TeleportInternalResourceType: types.SystemResource,
+				types.OriginLabel:                  common.OriginAWSIdentityCenter,
+			},
+		},
+		Spec: types.RoleSpecV6{
+			Allow: types.RoleConditions{
+				AccountAssignmentLabels: types.Labels{
+					types.OriginLabel: []string{common.OriginAWSIdentityCenter},
+				},
+				Rules: []types.Rule{
+					types.NewRule(types.KindIdentityCenter, RO()),
+				},
```

Comment:
Doe the RO rule for `KindIdentityCenter is really needs to access AWS IC resource ?

Reply:
Things stop working when I take it out, so... :-)

Looking at the implementation of `ListResources()`, the role access rules are explicitly checked, so I think it's necessary.